app/models/BusinessPremisesRenovationAllowanceBalancingCharge.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{Format, Json}
+
+final case class BusinessPremisesRenovationAllowanceBalancingCharge(
+  businessPremisesRenovationAllowanceBalancingChargeYesOrNo: Boolean,
```

Comment:
Can you use a simpler variable name? The class name already indicates what page this is.

Reply:
Yes, this has changed in the CYA FE-BE integration ticket to match what the BE expects